ibm_ace/tests/test_config.py

```diff
@@ -0,0 +1,72 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.ibm_ace import IbmAceCheck
+
+from .common import skip_windows_ci
+
+pytestmark = [skip_windows_ci, pytest.mark.usefixtures('dd_environment'), pytest.mark.unit]
+
+
+@pytest.mark.parametrize(
+    "test_instance, expected_res_stats, expected_msg_flows",
+    [
+        pytest.param(
+            {
```

Comment:
nit to make this shorter: we can define the instance in the test itself and override the `resource_statistics` and `message_flows params` with  the values from `expected_res_stats` and `expected_msg_flows`

Reply:
thanks! i'll do that